src/main/java/com/eppo/sdk/dto/AssignmentLogData.java

```diff
@@ -8,34 +8,24 @@
 public class AssignmentLogData {
     public String experiment;
     public String featureFlag;
-    public String assignmentModelVersion;
```

Comment:
Backtracking on changes from https://github.com/Eppo-exp/java-server-sdk-bandit-beta/pull/1 to put this in bandit log data

Reply:
Correct (and nice job spotting that)! The bandits team is currently leaning towards leaving the existing assignments log table alone and putting bandit stuff in specifically bandit-purposed tables. ([relevant design doc section](https://www.notion.so/eppo/Bandits-Engineering-Design-c5df5b6072b446aa8cacbf54e4d1f9df?pvs=4#263ada470ab34b8c91a906914788dd3f))

Note we'll still log the variation (e.g., `bandit`) to assignments when the bandit is selected, but it's action (e.g., `nike ad`) will be logged to the bandits table along with the variation and other useful metadata. Some data will be duplicated, but data is "cheap" 😅